src/core_write.cpp

```diff
@@ -224,9 +234,18 @@ void TxToUniv(const CTransaction& tx, const uint256& hashBlock, UniValue& entry,
         ScriptPubKeyToUniv(txout.scriptPubKey, o, true);
         out.pushKV("scriptPubKey", o);
         vout.push_back(out);
+
+        amt_total_out += txout.nValue;
```

Comment:
The fuzzer can trigger undefined behaviour here with an overflow, which causes CI to fail even though it's harmless. Not sure what the solution should be - seems like overkill to actually prevent.

Reply:
@luke-jr Functions are assumed to be robust as long as they are fed with inputs fulfilling their documented preconditions, so I think the only robust alternative to preventing it is to make the implicit precondition explicit by documenting it. However, in this case it seems easy to just use `MoneyRange(…)` and be done with it, no? :)